frontend/src/features/mapSideBar/hooks/useQueryParamSideBar.tsx

```diff
@@ -114,6 +123,17 @@ const useQueryParamSideBar = (): IMapSideBar => {
     [setShow, showSideBar],
   );
 
+  const handleLocationChange = (location: H.Location, action: 'PUSH' | 'POP' | 'REPLACE') => {
```

Comment:
these are all of the conditions that are considered "navigating away from the form"